PowerPointLabs/PowerPointLabs/PowerPointLabs.csproj

```diff
@@ -285,7 +285,6 @@
     <Compile Include="TextCollection.cs" />
     <Compile Include="Utils\Common.cs" />
     <Compile Include="Utils\Comparers.cs" />
-    <Compile Include="Utils\Extensions.cs" />

```

Comment:
Yes it is an intended change.

On Thursday, May 14, 2015, Damith C. Rajapakse <notifications@github.com>
wrote:

> In PowerPointLabs/PowerPointLabs/PowerPointLabs.csproj
> <https://github.com/PowerPointLabs/powerpointlabs/pull/703#discussion_r30322483>
> :
>
> > @@ -285,7 +285,6 @@
> >      <Compile Include="TextCollection.cs" />
> >      <Compile Include="Utils\Common.cs" />
> >      <Compile Include="Utils\Comparers.cs" />
> > -    <Compile Include="Utils\Extensions.cs" />
>
> Is this an intended change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PowerPointLabs/powerpointlabs/pull/703/files#r30322483>
> .
>


-- 
Best Regards,
Huang Da (Mr.)
National University of Singapore, Computer Engineering, Year 4
+65 86552730